packages/pds-ember/app/styles/pds/components/icon/__config.scss

```diff
@@ -3,22 +3,32 @@ $_module: "PDS.Components.Icon";
 
 $color: currentColor;
 
-@mixin appearance {
-  /* [debug] #{$_module}@appearance */
+@mixin layout {
+  /* [debug] #{$_module}@layout */
   display: inline-block;
   flex-shrink: 0;
   font-size: inherit;
   height: 1em;
-  line-height: 1;
+  line-height: inherit;
   width: 1em;
 
+  &__svgWrapper {
+    display: inline-block;
+    line-height: 1;
```

Comment:
This is the original `line-height` that existed before.   The override is necessary in order to keep SVG `<path>`s  from being shifted within the viewBox.